INSTALL.md

```diff
@@ -164,13 +164,21 @@ sudo dnf groupinstall "Development Tools"
 
 ##### macOS
 
-Installing prerequisites on macOS:
+Installing prerequisites on macOS (Big Sur):
```

Comment:
It's confusing that this says "Big Sur" here, but then below it talks about Catalina.

Reply:
I didn't delete the instructions for Catalina and didn't check those, but, we could delete those entirely, or, keep them a bit until Big Sur is confidently free of security patches. Not sure how that would typically be handled in those docs, do you know?

Reply:
Sorry for being unclear, my point was that I don't think we should add the ` (Big Sur)` here, we should just leave it as `macOS`.

Reply:
Fixed in the most recent commit.